plugins/sip/lib/model/providers/pexip/kPexipUtils.php

```diff
@@ -237,10 +237,13 @@ public static function retrieveAndValidateEntryForSipCall($queryParams, $pexipCo
 				return false;
 			}
 
-			$dbDualStreamLiveEntry = entryPeer::retrieveByPK($dualStreamEntryId);
+			$currentEntitlementEnforcement = kEntitlementUtils::getEntitlementEnforcement();
+                        kEntitlementUtils::initEntitlementEnforcement( $dbLiveEntry->getPartnerId() , false);
```

Comment:
Let call the following only if currentEntitlementEnforcement is true

to simplify it - 
!$currentEntitlementEnforcement || kEntitlementUtils::initEntitlementEnforcement( $dbLiveEntry->getPartnerId() , false);